lib/basic_types.ex

```diff
@@ -375,7 +374,7 @@ defmodule PropCheck.BasicTypes do
   """
   @spec ordered_list(raw_type()) :: type
   def ordered_list(elem_type) do
-    let l <- list(elem_type), do: :lists.sort(l)
+    let(l <- list(elem_type), do: :lists.sort(l))
```

Comment:
This looks really ugly, does'n it? It appears to me as if for macros always parantheses would be used. Such as 

    for(x <- xs, do: x+1) 

instead of 

    for x <- xs, do: x+1

Can this be handled nicer with the formatter? I never worked with, so I have no experience.

Reply:
Yep, I agree, and it can be configured.
So if we're ok with the idea of bringing the formatter in general, I can try to add some rules.